lib/spec_file_generator.js

```diff
@@ -119,6 +133,28 @@ function getSummary(description) {
   return description !== undefined ? description.split('.')[0].replace(/^\s+|\s+$/g, '') : 'FIXME';
 }
 
+function getProjectUrl(npm_module) {
+  let projecturl = '';
+
+  if (npm_module.homepage) {
+    projecturl = npm_module.homepage;
+  } else if (npm_module.repository) {
+    projecturl = npm_module.repository.url;
+  } else {
+    projecturl = "https://www.npmjs.com/package/" + npm_module.name;
+  }
+
+  return projecturl;
+}
+
+function getLicense(npm_module) {
```

Comment:
I like this, but can you add an `else` which returns `FIXME`? I've ran into packages without a license and then it generates an invalid spec file which causes `spectool` to fail later on.